test/functional/testCases.js

```diff
@@ -513,6 +513,399 @@ const testCases = [
             }
         ]
     },
+    {
+        describeName: '0022 - Simple group with active attributes and multimeasures',
+        provision: {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/services',
+            method: 'POST',
+            json: {
+                services: [
+                    {
+                        resource: '/iot/json',
+                        apikey: globalEnv.apikey,
+                        entity_type: globalEnv.entity_type,
+                        commands: [],
+                        lazy: [],
+                        attributes: [
+                            {
+                                object_id: 'a',
+                                name: 'attr_a',
+                                type: 'Number'
+                            }
+                        ],
+                        static_attributes: []
+                    }
+                ]
+            },
+            headers: {
+                'fiware-service': globalEnv.service,
+                'fiware-servicepath': globalEnv.servicePath
+            }
+        },
+        should: [
+            {
+                shouldName:
+                    'A - WHEN sending defined object_ids (measures) through http IT should send measures to Context Broker preserving value types and name mappings',
+                type: 'multimeasure',
```

Comment:
Note that `multimeasure` is not included in the documentation at https://github.com/telefonicaid/iotagent-node-lib/tree/master/test/functional/README.md

> `type`: The type of the test case. This can be `single` or `multientity`. See the "Advanced features" section for more information.

If we have invented a new type in this PR, I understand it should be properly documented there.

(I already published this comment, but it probably get lost :)

Reply:
Interestingly, there is a section about multimeasures in such documentation https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/functional/README.md#multimeasures, but the relation with the test `type` is not mentioned there... (probably it should)

Reply:
`multimeasures` was already supported for measures reception, but not for the expectations.

New changes were included since this PR changed the way of multimeasures are sent. In this case, the expectation changes. Thus, a new parameter is needed